app/javascript/components/app.js

```diff
@@ -19,11 +19,10 @@ const App = () => {
   return (
     <Routes>
 
-
       <Route path="/" element={isAuthenticated ? <Home /> : <Navigate to="/login" />} />
       <Route path="/login" element={<LoginPage />} />
       <Route path="/motorbikes/:id" element={isAuthenticated ? <Details /> : <Navigate to="/login" />} />
```

Comment:
The login is working very well, but not the way we expect it. We have to review it together, cause we need the current_user_id to make a reservation.